cfpb/consumerfinance.gov
# Remove unused legacy templatetags

This commit removes three unused legacy templatetags files. These tags are no longer used anywhere in this repository or in any cf.gov satellite apps.

- [get_static_version](https://github.com/search?q=user%3Acfpb+static_version&type=code) (also removed unused settings.STATIC_VERSION)
- [is_feature_active](https://github.com/search?q=user%3Acfpb+is_feature_active&type=code)
- [is_feature_view_active](https://github.com/search?q=user%3Acfpb+is_feature_view_active&type=code)
- [is_remit](https://github.com/search?q=user%3Acfpb+is_remit&type=code) (also removed related CSS classes)

## How to test this PR

Run a local server and confirm that legacy templates still function:

- http://localhost:8000/know-before-you-owe/
- http://localhost:8000/data-research/consumer-complaints/

## Checklist
